github/repositories/terraform-aws-modules/common.hcl

```diff
@@ -34,4 +34,87 @@ inputs = {
     }
 
   }
+
+  labels = {
+    "breaking change" = {
+      color       = "d66d0a"
+      description = "Breaking change features"
+    }
+
+    "bug" = {
+      color       = "d73a4a"
+      description = "Something isn't working"
+    }
+
+    "duplicate" = {
```

Comment:
not sure about this one. I feel like if its a duplicate, we just close it out and reference the open issue/PR

Reply:
I guess the same with `invalid`. We will just close it.